ippserver/server.py

```diff
@@ -115,6 +118,12 @@ def handle_www(self):
                 status=200, content_type='text/plain'
             )
             self.wfile.write(self.server.behaviour.ppd.text())
+        elif self.path.endswith('.png'):
+            self.send_headers(
+                status=200, content_type='image/png'
+            )
+            with open(local_file_location(self.path), 'rb') as wwwfile:
+                self.wfile.write(wwwfile.read())
```

Comment:
We should do something here to prevent self.path being some horror like `"../../etc/passwd"`. Assert that path is a regexp pattern, or something?